app/controllers/state_file/questions/nj_disabled_exemption_controller.rb

```diff
@@ -0,0 +1,17 @@
+module StateFile
+  module Questions
+    class NjDisabledExemptionController < QuestionsController
+      include ReturnToReviewConcern
+
+      before_action -> { @filing_year = Rails.configuration.statefile_current_tax_year }
+
+      def self.show?(intake)
+        if intake.filing_status_mfj?
+          !intake.direct_file_data.is_primary_blind? || !intake.direct_file_data.is_spouse_blind?
+        else
+          !intake.direct_file_data.is_primary_blind?
+        end
+      end
```

Comment:
NIT: I find this a little tough to parse as it's using inverse of booleans and short circuiting. You may consider being explicit and using multiple returns.

```ruby
if intake.filing_status_mfj?
   return false if intake.direct_file_data.is_primary_blind?
   #... snip
```

I think it may be useful to have a small comment indicating why this is inverting the value because it took me a second to understand that this was being used to indicate someone may have missed an opportunity to claim an exemption.

Reply:
Sounds good! Yea I'd approve that!

Reply:
Put out another revision! Had to update the logic a bit - the only case where we don't want to show the controller is if BOTH the primary and the spouse are blind, so I don't think we can separate out those clauses.